src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,52 @@
+package parser;
+
+import org.junit.After;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import shop.Cart;
+
+import java.io.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class JsonParserTest {
+
+    @Test
+    @Tag("ParserTests")
+    void testWriteToFile() {
+        JsonParser parser = new JsonParser();
+        parser.writeToFile(new Cart("testCartFile"));
+        File testCartFile = new File("src/main/resources/testCartFile.json");
+        assertTrue(testCartFile.exists());
+
+    }
+
+    @Test
+    @Disabled
+    @Tag("ParserTests")
+    void testReadFromFile() {
+        JsonParser parser = new JsonParser();
+        File predefinedCart = new File("src/main/resources/predefinedCart.json");
+        parser.readFromFile(predefinedCart);
```

Comment:
the main part of the test is assertion. In current state your test doesn't check anything